playbooks/pre.yaml

```diff
@@ -4,20 +4,20 @@
   roles:
     - role: ensure-pip  # https://zuul-ci.org/docs/zuul-jobs/latest/python-roles.html#role-ensure-pip
   tasks:
-    - name: Copy ADRs on the node
-      ansible.builtin.copy:
-        src: "../Standards"
-        dest: "~/"
-        mode: 0500
-      no_log: false
+    # - name: Copy ADRs on the node
```

Comment:
I don't quite understand this change, but more importantly, I'm quite sure that it will break the other jobs. I get that this PR is a draft, just my 2 cents

Reply:
I know, it was an interim step and I kept "current" state as pre-old.yaml with the intention to do similar improvement for other job separately.